src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -9869,98 +9869,135 @@ static bool isCandidateUnique(ref MethodSymbol? method, MethodSymbol candidate)
                 return GetUniqueSignatureFromMethodGroup_CSharp10(node);
             }
 
-            MethodSymbol? method = selectMethodForSignature(node);
-
-            if (method is null)
+            MethodSymbol? foundMethod = null;
+            var typeArguments = node.TypeArgumentsOpt;
+            if (node.ResultKind == LookupResultKind.Viable)
             {
-                return null;
-            }
+                foreach (var instanceMethod in node.Methods)
```

Comment:
>instanceMethod [](http://example.com/codeflow?start=29&length=14)

The name feels somewhat confusing. The method can be a static method, right? "memberMethod" instead?